config/environments/local.config.ts

```diff
@@ -13,7 +13,8 @@ export default {
         dm_store_api: 'https://dm-store-aat.service.core-compute-aat.internal',
         em_anno_api: 'https://em-anno-aat.service.core-compute-aat.internal',
         em_npa_api: 'https://em-npa-aat.service.core-compute-aat.internal',
-        coh_cor_api: 'https://coh-cor-aat.service.core-compute-aat.internal'
+        coh_cor_api: 'https://coh-cor-aat.service.core-compute-aat.internal',
+        dg_docassembly_api: 'http://localhost:4631'
```

Comment:
Is this because your doc-assembly container is exposing itself to your host machine on port 4631?

Reply:
yep